src/machine.js

```diff
@@ -245,10 +243,16 @@ class Machine {
   response(params: any): ResponseShape {
     return {
       current_state: this.getState(),
-      available_transitions: this.getTransitions(),
-      data: this.getData(params),
+      available_transitions: this.#state ? this.getTransitions(): [],
```

Comment:
this condition must be handled inside of `this.getTransitions`- where it returns an empty array if there is no state set.